proto/viam/app/data/v1/data.proto

```diff
@@ -54,6 +54,9 @@ service DataService {
 
   // BoundingBoxLabelsByFilter gets all string labels for bounding boxes from data based on given filter.
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
+
+  // GetDatabaseUser gets user credentials for any Viam user within their org to access a MongoDB Atlas Data Federation instance.
```

Comment:
```suggestion
  // GetDatabaseUser gets database user credentials for the given Viam organization to access a MongoDB Atlas Data Federation instance.
```

Nice! I modified this slightly since not all Viam users will have access to hit this API.